internal/providersdkv2/resource_hvn_route_test.go

```diff
@@ -16,90 +17,541 @@ import (
 
 var (
 	// using unique names for resources to make debugging easier
-	hvnRouteUniqueAWSName = fmt.Sprintf("hcp-provider-test-%s", time.Now().Format("200601021504"))
-	testAccHvnRouteConfig = fmt.Sprintf(`
-provider "aws" {
-  region = "us-west-2"
+	hvnRouteUniqueName       = fmt.Sprintf("hcp-provider-test-%s", time.Now().Format("200601021504"))
+	testAccHvnRouteConfigAws = fmt.Sprintf(`
+	  provider "aws" {
```

Comment:
Nit: These strings are a bit hard to read. Maybe they should be in a testdata folder?

Reply:
I agree! Seems this is the strategy throughout the provider. I'm planning a side-quest to see how we can make this better.